.github/workflows/release.yaml

```diff
@@ -0,0 +1,46 @@
+
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Turnstyle
+        uses: softprops/turnstyle@v1
+        with:
+          continue-after-seconds: 180
+        env:
+          GITHUB_TOKEN: ${{ secrets.CR_TOKEN }}
```

Comment:
I created `CR_TOKEN` but just curious if `secrets.GITHUB_TOKEN` could be used here.

Reply:
I believe it should be fine to use the `secrets.GITHUB_TOKEN` instead, I'll update the PR to reflect.  The `CR_TOKEN` reference comes from the [implementation of the chart releaser action](https://github.com/helm/chart-releaser-action/blob/master/cr.sh#L23) and must be set as a local environment variable named that in the action environment setting.

This does not mean, of course, that the _GitHub Secret_ must also be named the same.

Reply:
This change should now be present!